javapeople.yaml

```diff
@@ -298,6 +298,9 @@
 - name: "Joseph Darcy"
   twitter: "@jddarcy"
   fediverse: "@jddarcy@mastodon.social"
+- name: "Joseph Ottinger"
+  twitter: 'josephbottinger'
```

Comment:
Please insert @ before the twitter handle to make the links on the site work. Thx!